applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,2288 @@
+     */
+    loadFileList: function(path) {
+        var self = this;
+        selectedItems.clear(); // Clear selected items when loading a new file list
+
+        return getFileList(path).then(function(files) {
+            var fileList = document.getElementById('file-list');
+            if (!fileList) {
+                ui.addNotification(null, E('p', _('Failed to display the file list.')), 'error');
+                return;
+            }
+            fileList.innerHTML = ''; // Clear existing file list
+            files.sort(self.compareFiles.bind(self)); // Sort files based on the current sort field and direction
+
+            // Add a row for the parent directory if not at root
+            if (path !== '/') {
+                var parentPath = path.substring(0, path.lastIndexOf('/')) || '/';
+                var listItemUp = E('tr', {
+                    'data-file-path': parentPath,
+                    'data-file-type': 'directory'
+                }, [
+                    E('td', {
+                        'colspan': 5
+                    }, [
+                        E('a', {
+                            'href': '#',
+                            'click': function() {
+                                self.handleDirectoryClick(parentPath);
+                            }
+                        }, '.. (Parent Directory)')
+                    ])
+                ]);
+                fileList.appendChild(listItemUp);
+            }
+
+            // Iterate over each file and create table rows
+            files.forEach(function(file) {
+                var listItem;
+                var displaySize = (file.type === 'directory' || (file.type === 'symlink' && file.size === -1)) ? -1 : file.size;
+
+                // Create a checkbox for selection
+                var checkbox = E('input', {
+                    'type': 'checkbox',
+                    'class': 'select-checkbox',
+                    'data-file-path': joinPath(path, file.name),
+                    'change': function(ev) {
+                        self.handleCheckboxChange(ev); // Handle checkbox change event
+                    }
+                });
+
+                // Create an array for the action buttons
+                var actionButtons = [
+                    checkbox, // Add the checkbox to the Actions cell
+                    // Edit button
+                    E('span', {
+                        'class': 'edit-button',
+                        'click': function() {
+                            self.handleEditFile(joinPath(path, file.name), file);
+                        }
+                    }, '✏️'),
+                    // Duplicate button
+                    E('span', {
+                        'class': 'duplicate-button',
+                        'click': function() {
+                            self.handleDuplicateFile(joinPath(path, file.name), file);
+                        }
+                    }, '📄'),
```

Comment:
perhaps the two page emoji

Reply:
I didn't find 2 page emoji and replaced with this one '🔁'